.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
```

Comment:
love seeing more GH actions vs CircleCI 😍 . i've working on a side, when i have extra time, to move our infra/sre pre-commit circleci to GH actions too